stable/prometheus-blackbox-exporter/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Prometheus Blackbox Exporter
 name: prometheus-blackbox-exporter
-version: 3.2.0
+version: 3.2.1
```

Comment:
Since this is a new feature, could you bump a MINOR version instead of PATCH?

Reply:
done